src/Humanizer/InflectorExtensions.cs

```diff
@@ -22,153 +22,16 @@
 //CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 using System;
-using System.Collections.Generic;
 using System.Text.RegularExpressions;
+using Humanizer.Inflections;
 
 namespace Humanizer
 {
     /// <summary>
-    /// Provides hint for Humanizer as to whether a word is singular, plural or with unknown plurality
-    /// </summary>
-    public enum Plurality
```

Comment:
@jkodroff what is the reason, why you moved this enum? Please move it back to `Humanizer